### PR TITLE
Fix minor issues in contributing md.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,20 +28,21 @@ information on using pull requests.
 Fork the repo, checkout the upstream repo to your GOPATH by:
 
 ```
-$ go get -d github.com/census-instrumentation/opencensus-service
+$ GO111MODULE="" go get -d github.com/census-instrumentation/opencensus-service
```
Comment (songy23) [inline CONTRIBUTING.md]: GO111MODULE=on?

Comment (bogdandrutu) [reply CONTRIBUTING.md]: Actually that doesn't work for me, I have to explicitly disable go modules to download the repo. Can you give a try?

Comment (songy23) [reply CONTRIBUTING.md]: Yes same for me, I tried
```bash
$ GO111MODULE=on go get -d github.com/census-instrumentation/opencensus-service
```
There're no errors but the repo isn't downloaded. Setting `GO111MODULE=""` works for me.

Comment (bogdandrutu) [reply CONTRIBUTING.md]: Exactly, but I think the fact that the repo doesn't download is an error :)

**Pull request merged**